src/hooks/useAxiosWithAuth0.js

```diff
@@ -23,10 +23,25 @@ export default function useAxiosWithAuth0() {
   };
 
   useEffect(() => {
-    (async () => {
-      const token = await getAccessTokenSilently();
-      axiosWithAuth(token);
-    })();
+    const locations = [
```

Comment:
I like that you came up with the idea to organize all the public routes in one array, that was smart.